indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -397,6 +402,10 @@
             :access-value access-value
             :processing-level-id processing-level-id
             :processing-level-id-lowercase (util/safe-lowercase processing-level-id)
+            :latency latency
+            :latency-lowercase (util/safe-lowercase latency)
+            :latency-priority {:value latency
+                               :priority 0} 
```

Comment:
Why do we have a `priority` field if we don't actually use it?

Reply:
Most of our facets are using the priority one. I tried different kinds and couldn't make it work.